docs/mobility/base-rc/_index.md

```diff
@@ -140,10 +140,14 @@ Close out of all remote control related systems.
 For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/services/baseremotecontrol).
 
 ```go {class="line-numbers linkable-line-numbers"}
-baseRCService, err := baseremotecontrol.FromRobot(robot, "my_base_rc_service")
+baseRCService, err := baseremotecontrol.FromRobot(machine, "my_base_rc_service")
+if err != nil {
```

Comment:
We haven't done these checks in the rest of the docs. I think best to remove here?

Reply:
Oh interesting. Yeah alright then let's keep them. We could also consider adding error checking everywhere 🤷‍♀️

Reply:
Perfect